export const internalErrorOccurred = (message?: string) => {
	return "INTERNAL_ERROR"
}
